rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -587,6 +587,18 @@ def test_add_blockchain_accounts(
     })
     assert_error_response(response, 'appears multiple times in the request data')
 
+    # adding a taproot btc address
+    response = requests.put(api_url_for(
+        rotkehlchen_api_server,
+        'blockchainsaccountsresource',
+        blockchain='BTC',
+    ), json={'accounts': [
+        {'address': 'bc1pqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqsyjer9e'},
+    ]})
+
+    result = assert_proper_response_with_result(response)
+    assert result['per_account']['BTC'] is not None
```

Comment:
I think this would need a rebase since we have recently removed balances return from accounts addition